site/src/Controller/OrganizationController.php

```diff
@@ -215,6 +220,23 @@ public function joinAction($organizationName, OrganizationRepository $repository
         $member->setUser($user);
         $member->setPermission(OrganizationMember::PERMISSION_REQUEST);
         $em = $this->getDoctrine()->getManager();
+        // Find Organisation members
+        $orgMembers = $repository->findOneByName($organizationName)->getMemberUsers();
+        // Queue notifications to each member
+        $notification = new Notification();
+        foreach ($orgMembers as $recipient) {
+            // Prevent mailing the currentUser
+            if ($recipient !== $this->getUser()) {
+                $notification->setType("organisation_join")
+                    ->setSubject("Join organisation request")
+                    ->setRecipient($recipient)
+                    ->setMessage(
+                        $member->getUser()->getUsername()." has requested to join your organisation.
+                    Kindly go to the Librecores website to accept/reject the request."
```

Comment:
Please include the organization name, and a link to the org admin page (or directly link to the accept/reject actions). And please fix the indentation.